docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-azure.md

```diff
@@ -0,0 +1,172 @@
+```
+
+Replace values as follows:
+
+- `private_link_service_connection_name` - provide a name for the private link connection from the private endpoint to the private link service.
+- `private_endpoint_name` - provide a name for the private endpoint.
+- `pse_resource_id` - a property of your YBM PSE; for example, "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Network/privateLinkServices/MyPLS".
+- `resource_group_name` - the resource group in which the private endpoint will be created.
+- `subnet_id` - the ID of the subnet in the resource group in which the private endpoint will be created.
+- `private_endpoint_vnet_name` - the name of the Azure Virtual Network (VNet) where the private endpoint will be created.
+- `private_endpoint_region_name` - the Azure region in which the private endpoint and VNet are present.
+
+To ensure no change to your application settings and connect using sslmode=verify-full, create a private DNS zone in the same resource group and VNet as the private endpoint, and add an A record pointing to the IP address of the private endpoint.
+
+1. To create a private DNS zone, enter the following command:
+
+    ```sh
+    az network private-dns zone create \
+        --name <cluster_id>.azure.ybdb.io \
+        --resource-group <resource_group_name>
+    ```
+
+    Replace values as follows:
+
+    - `cluster_id` - the cluster ID of the cluster with the PSE; the cluster ID is displayed on the cluster **Settings** tab in YBM.
+    - `resource_group_name` - the resource group in which the private endpoint was created.
+
+1. To obtain the ipv4 address and VNet of the private endpoint, enter the following command:
+
+    ```sh
+    az network private-endpoint show \
+    --name <private_endpoint_name> \
+    --resource-group <resource_group_name>
+    ```
+
+    This command returns the following values that you use to link the private DNS zone to the VNET:
+
+    - ipv4 address of the private endpoint (`private_endpoint_ipv4_address` in the following commands)
+    - VNet of the private endpoint (`private_endpoint_vnet` in the following commands)
+
+1. To link the private DNS zone to the VNet containing the private endpoint, enter the following command:
+
+    ```sh
+    az network private-dns link vnet create 
+        --name <private_dns_zone_name>
+        --registration-enabled true
+        --resource-group <resource_group_name>
+        --virtual-network <private_endpoint_vnet>
+        --zone-name <cluster_id>.azure.ybdb.io
+        --tags yugabyte
+    ```
+
+    Replace values as follows:
+
+    - `private_dns_zone_name` - provide a name for the private DNS zone.
+    - `resource_group_name` - the resource group in which the private endpoint was created.
+    - `private_endpoint_vnet` - the VNet of the private endpoint.
+    - `cluster_id` - the cluster ID of the cluster with the PSE; the cluster ID is displayed on the cluster **Settings** tab in YBM.
```

Comment:
same comment as above -
"@ddhodge we will auto-generate this entire value as we know which cluster user is creating private endpoint for. I left it as <cluster_id> because this is based on the cluster. Can you please message this accordingly?"